torch/_inductor/ir.py

```diff
@@ -4772,15 +4772,8 @@ def _get_abi_compatible_kernel(self):
             return self.python_kernel_name
 
         def sdpa_ver_fn():
-            # For sdpa, we need the v2 version only if any optional
-            # kwarg is missing.
-            if any(
-                self.get_kwargs_value(arg_name) is None
-                for arg_name in self.ordered_kwargs_for_cpp_kernel
-            ):
-                return f"{self.cpp_kernel_name}_v2"
-            else:
-                return self.cpp_kernel_name
+            # For sdpa, we need the v2 version since v1 didn't consider optional arg
+            return f"{self.cpp_kernel_name}_v2"
 
         kernel_to_ver = {"at::_scaled_dot_product_flash_attention": sdpa_ver_fn}
```

Comment:
Since we only have a single version now, perhaps we could just get rid of the entire ```sdpa_ver_fn``` logic.